### PR TITLE
Add skip_assert_init macro calls to enums/flags codegen

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: src/codegen/enums.rs [hunk 1]
```diff
@@ -101,11 +101,13 @@ impl ToGlib for {name} {{
 }
 "));
 
+    let assert = if env.config.generate_safety_asserts { "skip_assert_init();\n\t\t" } else { "" };
```
Comment (EPashkin) [inline src/codegen/enums.rs]: `skip_assert_initialized!();`

File: src/codegen/flags.rs [hunk 2]
```diff
@@ -80,14 +80,16 @@ impl ToGlib for {name} {{
 }}
 ", name = flags.name, ffi_name = flags.c_type));
 
+    let assert = if env.config.generate_safety_asserts { "skip_assert_init();\n\t\t" } else { "" };
```
Comment (EPashkin) [inline src/codegen/flags.rs]: same

**Comment by gkoz** [comment]
Haha, it did make the script happy though!

**Comment by EPashkin** [comment]
May be we need fix script also? It second your PR with wrong skip macro.

**Comment by EPashkin** [comment]
Thanks. I wait appveyor and merge.

**Comment by EPashkin** [comment]
Strange, I get these errors:
```
src/auto/enums.rs: code(self) -> i32
```

**Comment by gkoz** [comment]
That's why I started fixing the script.

**Comment by EPashkin** [comment]
With `gir-asserts` in gtk it clean.

**Comment by EPashkin** [comment]
Travis good, so I merge. Thanks.

**Pull request merged**